### PR TITLE
chore: move shipping constants into cohesion

### PR DESCRIPTION
**Pull request by pepopowitz**
The type of this PR is: Chore

### Description

Adds constants related to the new shipping flow. We hardcoded [a couple of them in force](https://github.com/artsy/force/blob/e1a53b84f651d63990510df21f6d9ec69486bab3/src/v2/Apps/Order/Routes/Review/index.tsx#L440-L441) to get things out the door; this PR moves them to the correct home. 

It also includes several constants not being used yet in force, but which will allow us to specify the context module/owner type for every instance the Buyer Guarantee is clicked. We're currently only tracking those clicks from the shipping & review pages of the order app, but the component lives on pretty much every page in the order app.

### PR Checklist (tick all before merging)

- [x] If I've added a new file to the tree I've exported it from the common `index.ts`
- [x] I've added comments with examples for any new interfaces and ensured that they're in the docs
- [x] No platform-specific terminology has been used outside of `click` and `tap` (platform is inferred by the DB storing events)

### CODE REVIEW

File: src/Schema/Values/OwnerType.ts [hunk 1]
```diff
@@ -40,6 +40,14 @@ export enum OwnerType {
...
   myCollectionArtwork = "myCollectionArtwork",
   onboarding = "onboarding",
+  ordersAccept = "orders-accept",
+  ordersCounter = "orders-counter",
+  ordersNewPayment = "orders-new-payment",
+  ordersOffer = "orders-offer",
+  ordersPayment = "orders-payment",
+  ordersRespond = "orders-respond",
+  ordersReview = "orders-review",
+  ordersShipping = "orders-shipping",
```
Comment (pepopowitz) [inline src/Schema/Values/OwnerType.ts:50]: We are using these kebab-case values in the hardcoded force values, but everything else in this file is using camelCase. Is that a problem???

File: src/Schema/Values/ContextModule.ts [hunk 2]
```diff
@@ -86,8 +86,14 @@ export enum ContextModule {
   marketInsights = "marketInsights",
   newWorksByArtistsYouFollowRail = "newWorksByArtistsYouFollowRail",
   newWorksByGalleriesYouFollowRail = "newWorksByGalleriesYouFollowRail",
-  ordersShipping = "ordersShipping",
```
Comment (damassi) [inline src/Schema/Values/ContextModule.ts:89]: Can alphabetize this? (noted that it was out of alpha to begin with)

Comment (pepopowitz) [reply src/Schema/Values/ContextModule.ts:89]: ugh, I thought I did but apparently I don't know the English alphabet 😅 

👍 Will do.

Comment (pepopowitz) [reply src/Schema/Values/ContextModule.ts:89]: Resolved in most recent commit!

**Comment by pepopowitz** [comment]
@brendamanganelli could you provide a review for this so that I can move forward with it on the Force side?

File: src/Schema/Values/OwnerType.ts (hunk 1 above)
Comment (brendamanganelli) [reply src/Schema/Values/OwnerType.ts:50]: This is great and it is consistent with what we've been doing!

**Review by brendamanganelli — APPROVED** [review]
No objections on my side! Ship it!

**Pull request merged**